app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt

```diff
@@ -11,25 +12,35 @@ import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import androidx.recyclerview.widget.DividerItemDecoration
 import androidx.recyclerview.widget.LinearLayoutManager
-import kotlinx.android.synthetic.main.fragment_favorite.noLikedText
-import kotlinx.android.synthetic.main.fragment_favorite.view.favoriteEventsRecycler
-import kotlinx.android.synthetic.main.fragment_favorite.view.favoriteProgressBar
+import kotlinx.android.synthetic.main.fragment_favorite.*
+import kotlinx.android.synthetic.main.fragment_favorite.view.*
```

Comment:
Wildcard imports are disallowed. I don't even know why the build didn't fail. Let me check and fail it

Reply:
Removed